.editorconfig

```diff
@@ -0,0 +1,13 @@
+; EditorConfig is awesome: http://EditorConfig.org
+
+; top-most EditorConfig file
+root = true
+
+; base rules
+[*]
```

Comment:
I use this with Vim so it can automatically detect tabs/spaces, etc. I think this covers the conventions used on the project. If not, I can tweak it or remove it if you don't want it.

Reply:
I guess I should check in the code formatter files for Eclipse.

Reply:
I was looking for them initially before going this route which is still helpful I think.